packages/react-flicking/src/react-flicking/index.tsx

```diff
@@ -162,6 +156,10 @@ export default class Flicking extends React.Component<IFlickingProps> {
   public getPrevIndex(physical: boolean) {
     return this.flicking.getPrevIndex(physical);
   }
+  public plugin(plugins: any[]) {
```

Comment:
@younkue How about to provide plugin interface instead of `any`

Reply:
@jongmoon has no type in ```@egjs/flicking-plugins```